lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,48 @@
+<div class="cohort-discussions-nav">
+    <h3 class="subsection-title"><%- gettext('Specify whether discussion topics are divided by cohort') %></h3>
+    <form action="" method="post" id="cohort-coursewide-discussions-form" class="cohort-coursewide-discussions-form">
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="coursewide-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Course-wide Discussion Topics') %></h3>
+                        <p><%- gettext('Select the course-wide discussion topics that you want to divide by cohort.') %></p>
+                        <div class="field">
+                            <ul class="discussions-wrapper"><%= coursewideTopics %></ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="form-actions">
+                <button class="form-submit button action-primary action-save"><%- gettext('Save') %></button>
+            </div>
+        </div>
+    </form>
+
+    <hr class="divider divider-lv1" />
+
+    <form action="" method="post" id="cohort-inline-discussions-form" class="cohort-inline-discussions-form">
+        <div class="wrapper cohort-management-supplemental">
+            <div class="form-fields">
+                <div class="form-field">
+                    <div class="inline-discussion-topics">
+                        <h3 class="subsection-title"><%- gettext('Content-Specific Discussion Topics') %></h3>
+                        <p><%- gettext('Select the content-specific discussion topics that you want to divide by cohort. To specify that all existing and future content-specific discussion topics are divided by cohort, select ') %><b><%- gettext('Always cohort content-specific discussion topics') %></b>.</p>
```

Comment:
Per conversations on this PR: replace this UI text with "Specify whether content-specific discussion topics are divided by cohort." 
along with new radio buttons